pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -22,7 +22,7 @@ in pythonPackages.buildPythonApplication rec {
       --replace 'self.init_system = ""' 'self.init_system = "systemd"'
 
     substituteInPlace cloudinit/config/cc_growpart.py \
-      --replace 'util.subp(["growpart"' 'util.subp(["${cloud-utils}/bin/growpart"'
+      --replace 'util.subp(["growpart"' 'util.subp(["${e2fsprogs}/bin/growpart"'
```

Comment:
Does e2fsprogs actually contain growpart?
```
$ nix-build -A e2fsprogs .
/nix/store/q5vqv079srs8y131qgsqnh4422azc013-e2fsprogs-1.43.9-bin
$ find /nix/store/q5vqv079srs8y131qgsqnh4422azc013-e2fsprogs-1.43.9-bin/ -name growpart
$ # no results
```

(Nixpkgs version: 6c064e6b1f34a8416f990db0cc617a7195f71588 )

Reply:
hmmm... arf. I did a mistake somewhere. `growpart` doesn't come with `e2fsprogs`. So I close this PR and my apologies...